gradle/libs.versions.toml

```
@@ -1,8 +1,8 @@
 [versions]
 # ArcGIS Maps SDK for Kotlin version
-arcgisMapsKotlinVersion = "200.4.0-4121"
+arcgisMapsKotlinVersion = "200.4.0-4189"
 # ArcGIS Maps SDK for Kotlin Toolkit version
-arcgisToolkitVersion = "200.4.0-4121"
+arcgisToolkitVersion = "200.4.0-4183"
```

Let's use the latest toolkit build here: 
```suggestion
arcgisToolkitVersion = "200.4.0-4189"
```